ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -209,6 +209,10 @@ QtObject {
         }
     }
 
+    function canProfileProveOwnershipOfProvidedAddresses(addresses) {
+        return walletSection.canProfileProveOwnershipOfProvidedAddresses(JSON.stringify(addresses))
+    }
```

Comment:
The accounts model contains `keycardAccount` role, maybe we could also provide role determining if given keypair can be proved by profile, then ui could check it via model with not need for special method for that :thinking:

Reply:
Well, any keypair that is not migrated to a keycard can be proved by profile keypair. So in this context, we can expose a bool property like `keypairMigratedToKeycard` and do the check on the qml side, so far the logic is on the Nim side.

Reply:
Ok, for now it's just a question, we can consider changes later